bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html

```diff
@@ -304,7 +304,7 @@ <h2 uitestid="ktb-trigger-headline">Trigger an evaluation for {{ selectedService
           >
             <dt-icon name="calendar"></dt-icon>
           </button>
-          <dt-hint>Has to be after start date</dt-hint>
+          <dt-hint>The end date and time of an evaluation that has to be after the start date and time.</dt-hint>
```

Comment:
```suggestion
          <dt-hint>The end date and time of an evaluation that must be after the start date and time.</dt-hint>
```

Reply:
Thanks for the hint! I changed it accordingly.